lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -348,7 +349,7 @@ export class AuthorizationCodeClient extends BaseClient {
         );
         parameterBuilder.addThrottling();
 
-        if (this.serverTelemetryManager) {
+        if (this.serverTelemetryManager && !(this.config.authOptions.authority.options.protocolMode === ProtocolMode.OIDC)) {
```

Comment:
nit: This deep check on the config object is a bit hard to visually parse in the middle of a bunch of other checks, maybe create a helper method to refactor this check out.

```
if (this.serverTelemetryManager && !oidcModeEnabled(this.config))
```

Where to put the function is up to you if you decide to go with it.